dist/angular-ui-tree.js

```diff
@@ -901,6 +910,11 @@
               //Get bounds of document.
               document_height = Math.max(body.scrollHeight, body.offsetHeight, html.clientHeight, html.scrollHeight, html.offsetHeight);
               document_width = Math.max(body.scrollWidth, body.offsetWidth, html.clientWidth, html.scrollWidth, html.offsetWidth);
+
+              if (scrollContainerElm) {
```

Comment:
Could you add a comment here what does it mean for scrollContainerElm to be defined vs not defined for the line below